build.yaml

```diff
@@ -0,0 +1,22 @@
+builders:
+  resourceBuilder:
+    import: "../../../tool/builder.dart"
+    builder_factories: ["resourceBuilder"]
+    build_extensions: {'$lib$': ['src/html/resources.g.dart']}
+    build_to: "source"
+    auto_apply: none
+
+targets:
```

Comment:
It's unusual to have a target with no sources. I wonder if we should consider this structure:

```yaml
targets:
  builder:
    sources: ["tool/builder.dart"]
  $default:
    sources:
      exclude: ["tool/builder.dart"]
    builders: {"dartdoc|resourceBuilder": {enabled: true}}
```